deployment/environments/dev/values.yaml

```diff
@@ -0,0 +1,4 @@
+contractAddress: 0xBC4CA0EdA7647A8aB7C2061c2E118A18a936f13D
+rpc: https://eth.llamarpc.com
+debug: false
+startingBlock: 41375688
```

Comment:
If you follow my suggestions of both helmfile.yaml, replace 41375688 with "41375688", otherwise the value will be passed as: 4.1375688e+07